wiki/Rules/de.md

```diff
@@ -45,7 +45,7 @@ Diese Regeln beziehen sich ausschließlich auf Beatmaps, die über das [Beatmap
 1. **Stelle sicher, dass du die Berechtigung hast, jeden Inhalt in deiner Beatmap zu verwenden.** Das umschließt Lieder, Videos, Hitsounds, Bilder oder andere Inhalte, die nicht von dir direkt selbst erstellt wurden. Solltest du dir unsicher sein, wo du frei nutzbare Inhalte finden kannst, kannst du die Bibliothek von Liedern unserer [Featured Artists](https://osu.ppy.sh/beatmaps/artists/) durchsuchen. Dort findest du Musik die zu 100% für die Benutzung in osu! lizensiert ist. Achte außerdem auf die [Regeln für die Inhaltsverwendung](Content_Usage_Guidelines).
 2. **Komm in [Beatmapdiskussionen](/wiki/Beatmap_Discussion) nicht vom Thema ab.** Die Threads und Diskussionen gelten der Beatmap und sonst nichts. Wenn du ein Problem mit etwas hast, was sich nicht direkt auf die Beatmap bezieht, dann veröffentliche es stattdessen im richtigen Forum.
 3. **Plagiiere oder stehle nicht die Arbeit anderer Nutzer.** Lade keine Inhalte anderer Nutzer ohne deren expliziten Erlaubnis hoch, einschließlich Skins und Guest Difficulties.
-4. **Folge den [Regeln für Liedinhalte](/wiki/Song_Content_Rules) und den [Empfehlungen für visuelle Inhalte](/wiki/Rules/Visual_Content_Considerations).**
+4. **Folge den [Regeln für Liedinhalte](Song_Content_Rules) und den [Empfehlungen für visuelle Inhalte](Visual_Content_Considerations).**
```

Comment:
What's the case where the `/wiki/` prefix is not required? ie. why is it removed here but not in `2.` above?

Reply:
these two as changed are relative links (`Song_Content_Rules` and `Visual_Content_Considerations` are in the `Rules` folder). `/wiki/` means absolute link, so as it was, it required a redirect.

The only reason this was reported was because of the redirect in `redirect.yaml` having the wrong casing (see 1b6f4b9), so it incurred two redirects. I left this fix on this line here while at it.

Reply:
Not sure about relative links in general, I think absolute reads better, but I guess that discussion is one to be had elsewhere.